tests/behavior/transforms/filter.toml

```diff
@@ -0,0 +1,67 @@
+[transforms.filter_a]
+  inputs = ["ignored"]
+  type = "filter"
+  [transforms.filter_a.condition]
+    "message.eq" = "test filter 1"
+[transforms.filter_b]
+  inputs = ["ignored"]
+  type = "filter"
+  [transforms.filter_b.condition]
+    type = "check_fields"
+    "message.contains" = "test filter"
+    "message.contains" = "2"
+[transforms.filter_c]
+  inputs = ["ignored"]
+  type = "filter"
+  [transforms.filter_c.condition]
+    type = "is_metric"
+
+[[tests]]
+  name = "filter test 1a"
+  no_outputs_from = [ "filter_b", "filter_c" ]
+  [[tests.inputs]]
+    insert_at = "filter_a"
+    value = "test filter 1"
+  [[tests.outputs]]
+    extract_from = "filter_a"
+    [[tests.outputs.conditions]]
+      "message.equals" = "test filter 1"
+
+[[tests]]
+  name = "filter test 1b"
+  no_outputs_from = [ "filter_a", "filter_b", "filter_c" ]
```

Comment:
```suggestion
  no_outputs_from = [ "filter_b" ]
```

Reply:
Just covering all the bases, since I saw other behavior tests doing similar.

Reply:
Oh ok :) I just found it very confusing.